tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/VersionConflict/AspNetVersionConflictTests.cs

```diff
@@ -110,7 +110,7 @@ public async Task Sampling(bool parentTrace)
             {
                 var samplingPriority = string.Empty;
 
-                if (span.Metrics.ContainsKey(Metrics.SamplingPriority))
+                if (span.Metrics != null && span.Metrics.ContainsKey(Metrics.SamplingPriority))
```

Comment:
Just for awareness, this also works 🙂 
```suggestion
                if (span.Metrics?.ContainsKey(Metrics.SamplingPriority) == true)
```